### ISSUE TITLE
Out of date supported DBMS list on website

### ISSUE THREAD

**Issue by MMatten** [dbfit/dbfit#560]
List of supported RDBMSs at http://dbfit.github.io/dbfit/ is out of date (missing Teradata, Netezza and Informix).

Is this site due a general revamp too?

**Comment by javornikolov** [comment]
> List of supported RDBMSs at http://dbfit.github.io/dbfit/ is out of date (missing Teradata, Netezza and Informix).

Indeed, good catch. The complete list of supported databases is here: http://dbfit.github.io/dbfit/docs/getting-started.html. But somehow we missed to update the title page.

> Is this site due a general revamp too?

Perhaps there is a room for improvement.

**Comment by SpokeyWheeler** [comment]
So is Informix supported or not? Because it's not in the complete list of supported databases.

**Comment by MMatten** [comment]
"it depends".

It's not in the 3.2.0 release.

It is in the `master` branch codebase (i.e. the still unreleased v4.0).
